lib/validations/disallowed_characters.rb

```diff
@@ -0,0 +1,17 @@
+class DisallowedCharacters < Grape::Validations::Base
```

Comment:
I'm wondering if we really need a custom validation class for this. It seems much cleaner to continue using Grape's built-in regexp validation. If the purpose of the custom validator was to provide more information in the error message, we could a custom validation message such as:
`regexp: { value: /^[a-z]+$/, message: "document_id is invalid: cannot contain any of the following characters: ['/'] " }`

Reply:
@MothOnMars I did indeed introduce the custom validator for the sole purpose of providing a more specific error message. I'm on the fence as to whether the regex is cleaner than the custom validator, but I made the change back to using a regex (this time disallowing all slashes instead of just leading slashes).

Reply:
Thanks. Adding an extra class & spec just felt like we were rebuilding a wheel already provided by Grape.